dropwizard/service/build.gradle.kts

```diff
@@ -183,4 +183,13 @@ tasks.register<Sync>("prepareDockerDist") {
 
 tasks.named("build").configure { dependsOn("prepareDockerDist") }
 
+distributions {
```

Comment:
Hm - it looks like none of the dist(s) actually works.

```
./gradlew :polaris-dropwizard-service:assembleDist
```
generates TWO sets of distributions:
```
$ ls -1 dropwizard/service/build/distributions/
polaris-dropwizard-service-1.0.0-incubating-SNAPSHOT-license-report.zip
polaris-dropwizard-service-1.0.0-incubating-SNAPSHOT.tar
polaris-dropwizard-service-1.0.0-incubating-SNAPSHOT.zip
polaris-dropwizard-service-shadow-1.0.0-incubating-SNAPSHOT.tar
polaris-dropwizard-service-shadow-1.0.0-incubating-SNAPSHOT.zip
```

The "non shadow" one has the NOTICE & LICENSE files, but running `polaris-dropwizard-service-1.0.0-incubating-SNAPSHOT/bin/polaris-service` leads to 
```
Error: Could not find or load main class org.apache.polaris.service.dropwizard.PolarisApplication
Caused by: java.lang.ClassNotFoundException: org.apache.polaris.service.dropwizard.PolarisApplication
```

The "shadow" does not have the NOTICE + LICENSE files, but running `polaris-dropwizard-service-shadow-1.0.0-incubating-SNAPSHOT/bin/polaris-dropwizard-service` does something.

Reply:
Yes, the `polaris-service` script never worked (see #542 ). It's not related to this PR, it's like this for a while.

Good point about "shadow" distribution. I actually wonder if you need both (only one binary distribution is enough).

Reply:
Frankly speaking, I think it's even better to get #469 in and let it replace the DW stuff entirely. Especially given the amount of issues.